### ISSUE TITLE
Signed USB device driver for Windows causes kernel panics ("blue screen of death") under Windows 10 x64

### ISSUE THREAD

**Issue by c0d3h4x0r** [retrogamehandheld/RetroXFW#4]
Using the USB connection method to a Windows 10 x64 PC is very risky because the device driver often causes BSODs ("blue screens of death") that prevent Windows from shutting down cleanly.  This can potentially corrupt the Windows filesystem and prevent Windows from booting or working correctly.  Please fix the device driver to stop causing blue screens.

**Comment by Ninoh-FOX** [comment]
In my win10 system work fine, do you have the correct drivers?

**Comment by c0d3h4x0r** [comment]
Yes.  The problem is intermittent.  It appears to be triggered by device disconnection/reconnection.  Just plug/unplug the RG350 a bunch of times and the system eventually bluescreens.  And because the USB-C connector on the RG350 is a bit loose, wiggling the cable just a little is enough to cause disconnection/reconnection.  Once I connect it, I have to be sure to not even breath on the cable, otherwise I risk getting a BSOD.

**Comment by Ninoh-FOX** [comment]
I think that is the cable, to my, if I don't have conect the console, the win10 said unknown usb device, whit only the cable!! But if I connect the console all work fine.

**Comment by c0d3h4x0r** [comment]
The problem is not the cable.  I've tried 4 different USB-C cables, all with the same results.

**Comment by sneak-thief** [comment]
Same thing happens to me occasionally when disconnecting the cable when Filezilla is first trying to login to the RG350.
